stabilizer/src/main/java/com/hazelcast/stabilizer/worker/tasks/SingleOperationAbstractWorkerTask.java

```diff
@@ -0,0 +1,33 @@
+package com.hazelcast.stabilizer.worker.tasks;
+
+/**
+ * Single operation version of {@link AbstractWorkerTask}.
+ * <p/>
+ * There is no {@link com.hazelcast.stabilizer.worker.selector.OperationSelector} used in this worker.
+ */
+public abstract class SingleOperationAbstractWorkerTask extends AbstractWorkerTask {
+
+    public SingleOperationAbstractWorkerTask() {
```

Comment:
Should we keep `Abstract` in names of abstract classes or not? We have `AbstractWorkerTask` and `AsyncAbstractWorkerClass`. Just renaming `SingleOperationAbstractWorkerTask` to something without `Abstract` is inconsistent :(